Client/Frontend/SurveySurface/SurveySurfaceViewController.swift

```diff
@@ -246,3 +236,20 @@ class SurveySurfaceViewController: UIViewController, Themeable {
         dismissSurveyButton.backgroundColor = theme.colors.actionSecondary
     }
 }
+
+// MARK: - UIViewController setup
+extension SurveySurfaceViewController {
```

Comment:
nit, we changed how we manage extension a couple months ago - this can be in the main class rather than an extension.

Reply:
Moved it to the main class, please have a look.